proof/refine/ARM/Schedule_R.thy

```diff
@@ -319,13 +318,13 @@ lemma setQueue_ksReadyQueues_lift:
   by (wp, clarsimp simp: fun_upd_def snd_def)
 
 lemma tcbSchedDequeue_valid_queues'[wp]:
-  "\<lbrace>valid_queues' and tcb_at' t\<rbrace>
+  "\<lbrace>valid_queues'\<rbrace>
     tcbSchedDequeue t \<lbrace>\<lambda>_. valid_queues'\<rbrace>"
```

Comment:
Very low priority, but I'm a fan of fixing up the indentation and layout of lemmas when I'm already touching them. Quite a few of the lemmas in this PR were previously wrong.